src/main/java/com/amihaiemil/zold/RestfulZoldWts.java

```diff
@@ -35,8 +35,7 @@
  * @author Mihai Andronache (amihaiemil@gmail.com)
  * @version $Id$
  * @since 0.0.1
- * @todo #4:30min Add methods for API entry points to this class, once we find
- *  the actual API docs for Zold.
+ * @todo #4 implement body of interface methods.
```

Comment:
@ammaratef45 This puzzle should start with ``@todo #11:30min ...`` (ticker #11 is the parent) and it should have a longer description, otherwise ``0pdd`` will complain (it needs to have at least 15 words, I think).